bundles/org.openhab.binding.sbus/README.md

```diff
@@ -57,13 +60,30 @@ The Sbus Bridge has the following configuration parameters:
 | id      | integer | Device ID of the switch controller                   | N/A     | yes      | no        |
 | refresh | integer | Refresh interval in seconds                          | 30      | no       | yes       |
 
+### Contact Sensor Configuration
+
+| Name    | Type    | Description                                          | Default | Required | Advanced |
+|---------|---------|------------------------------------------------------|---------|----------|-----------|
+| subnetId| integer | Subnet ID the contact sensor is part of              | N/A     | yes      | no        |
+| id      | integer | Device ID of the contact sensor                      | N/A     | yes      | no        |
+| refresh | integer | Refresh interval in seconds                          | 30      | no       | yes       |
+
```

Comment:
Maybe refactor this paragraph as Contact, Switch, Temperature all have the same configuration.